api/tests/Api/Invitations/FindInvitationTest.php

```diff
@@ -29,7 +29,7 @@ public function testFindInvitationWhenNotLoggedIn() {
         $this->assertResponseStatusCodeSame(200);
         $this->assertJsonContains([
             'campId' => $campCollaboration->camp->getId(),
-            'campTitle' => $campCollaboration->camp->title,
+            'campTitle' => $campCollaboration->camp->title ?: $campCollaboration->camp->shortname,
```

Comment:
Didn't we say that we always display shortname, and if we have space we additionally display campTitle?